### ISSUE TITLE
symfony-installer 1.4.1 fails to create Symfony 3 projects

### ISSUE THREAD

**Issue by MarcelHernandez** [symfony/symfony-installer#210]
Since it still looks for app/SymfonyRequirements.php, which no longer exists. Output:

```
$ symfony new project

 Downloading Symfony...

    4.93 MB/4.93 MB ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓  100%

 Preparing project...

PHP Warning:  require(/var/workspace/project/app/SymfonyRequirements.php): failed to open stream: No such file or directory in phar:///usr/local/bin/symfony/src/Symfony/Installer/DownloadCommand.php on line 265
PHP Stack trace:
PHP   1. {main}() /usr/local/bin/symfony:0
PHP   2. require() /usr/local/bin/symfony:10
PHP   3. Symfony\Component\Console\Application->run() phar:///usr/local/bin/symfony/symfony:35
PHP   4. Symfony\Component\Console\Application->doRun() phar:///usr/local/bin/symfony/vendor/symfony/console/Application.php:126
PHP   5. Symfony\Component\Console\Application->doRunCommand() phar:///usr/local/bin/symfony/vendor/symfony/console/Application.php:195
PHP   6. Symfony\Component\Console\Command\Command->run() phar:///usr/local/bin/symfony/vendor/symfony/console/Application.php:878
PHP   7. Symfony\Installer\NewCommand->execute() phar:///usr/local/bin/symfony/vendor/symfony/console/Command/Command.php:259
PHP   8. Symfony\Installer\DownloadCommand->checkSymfonyRequirements() phar:///usr/local/bin/symfony/src/Symfony/Installer/NewCommand.php:65
PHP Fatal error:  require(): Failed opening required '/var/workspace/project/app/SymfonyRequirements.php' (include_path='.:/usr/share/php:/usr/share/pear') in phar:///usr/local/bin/symfony/src/Symfony/Installer/DownloadCommand.php on line 265
PHP Stack trace:
PHP   1. {main}() /usr/local/bin/symfony:0
PHP   2. require() /usr/local/bin/symfony:10
PHP   3. Symfony\Component\Console\Application->run() phar:///usr/local/bin/symfony/symfony:35
PHP   4. Symfony\Component\Console\Application->doRun() phar:///usr/local/bin/symfony/vendor/symfony/console/Application.php:126
PHP   5. Symfony\Component\Console\Application->doRunCommand() phar:///usr/local/bin/symfony/vendor/symfony/console/Application.php:195
PHP   6. Symfony\Component\Console\Command\Command->run() phar:///usr/local/bin/symfony/vendor/symfony/console/Application.php:878
PHP   7. Symfony\Installer\NewCommand->execute() phar:///usr/local/bin/symfony/vendor/symfony/console/Command/Command.php:259
PHP   8. Symfony\Installer\DownloadCommand->checkSymfonyRequirements() phar:///usr/local/bin/symfony/src/Symfony/Installer/NewCommand.php:65

$ ls -l project/app/
total 20
-rw-r--r-- 1 marcel marcel  101 nov 30 22:15 AppCache.php
-rw-r--r-- 1 marcel marcel 1629 nov 30 22:15 AppKernel.php
-rw-r--r-- 1 marcel marcel  325 nov 30 22:15 autoload.php
drwxr-xr-x 2 marcel marcel 4096 dic  1 00:17 config
drwxr-xr-x 3 marcel marcel 4096 nov 30 22:15 Resources

$ head -n20 project/composer.json 
{
    "name": "marcel/project",
    "license": "proprietary",
    "type": "project",
    "autoload": {
        "psr-4": {
            "": "src/"
        },
        "files": [
            "app/AppKernel.php"
        ]
    },
    "autoload-dev": {
        "psr-4": {
            "Tests\\": "tests/"
        }
    },
    "require": {
        "php": ">=5.5.9",
        "symfony/symfony": "3.0.*",
```

**Comment by ogizanagi** [comment]
@MarcelHernandez : You can already update your install using `symfony selfupdate`. 
This has been fixed in #206 but unfortunately introduced another bug regarding `symfony demo` command: #208 :/

**Comment by MarcelHernandez** [comment]
Duh, sorry @ogizanagi , I didn't notice you already reported this issue. I'll close mine.
